python-sdk/tests_integration/integration_test_dag.py

```diff
@@ -100,7 +101,8 @@ def run_merge(output_table: Table):
         columns={"list": "list", "sell": "sell"},
         if_conflicts="ignore",
     )
-    con1 >> merged_table  # skipcq PYL-W0104
+
+    con1 >> merged_table >> aql.drop_table(merge_table)  # skipcq PYL-W0104
```

Comment:
Why are you dropping the table instead of running aql.cleanup()?

Reply:
yes, cleanup should take care of deleting them.